include-build/roborazzi-core/src/commonJvmMain/kotlin/com/github/takahirom/roborazzi/CaptureResults.kt

```diff
@@ -19,6 +19,73 @@ data class CaptureResults(
     return json
   }
 
+  fun toHtml(reportDirectoryPath: String): String {
+    fun File.pathFrom(reportDirectoryPath: String): String {
+      val reportDirectory = File(reportDirectoryPath)
+      val relativePath = relativeTo(reportDirectory)
+      return relativePath.path
+    }
+
+    val recordedImages = captureResults.filterIsInstance<CaptureResult.Recorded>()
+    val addedImages = captureResults.filterIsInstance<CaptureResult.Added>()
+    val changedImages = captureResults.filterIsInstance<CaptureResult.Changed>()
+    val unchangedImages = captureResults.filterIsInstance<CaptureResult.Unchanged>()
+    fun buildTable(title: String, images: List<CaptureResult>): String {
+      if (images.isEmpty()) return ""
+      return buildString {
+        append("<h3>$title (${images.size})</h3>")
+        val fileNameClass = "flow-text col s3"
+        val fileNameStyle = "word-wrap: break-word; word-break: break-all;"
+        val imgClass = "col s7"
+        val imgAttributes = "style=\"width: 100%; height: 100%; object-fit: cover;\""
+        append("<table class=\"highlight\">")
+        append("<thead>")
+        append("<tr class=\"row\">")
+        append("<td class=\"$fileNameClass\" style=\"$fileNameStyle\">File Name</td>")
+        append("<td class=\"$imgClass flow-text \">Image</td>")
```

Comment:
It's not important, but perhaps using the `<th>` element, which defines a header cell in an HTML table, would be more suitable here
https://www.w3schools.com/tags/tag_th.asp